constraints.txt

```diff
@@ -1,26 +1,27 @@
-boto3==1.16.60
-botocore==1.19.60
-cachetools==4.2.1
+boto3==1.24.27
```

Comment:
I already made changes like this in my CASMCMS-7957 PR. Why are you making this PR against develop as opposed to the release/1.3.0 branch? I realize in creating the release branch that I may not have told you or Ryan that I was doing so.